src/AppMain.java

```diff
@@ -59,4 +62,7 @@ public static void printValue(int value) {
         else comparison = false;
```

Comment:
Use another method for this function - [click here for the right code](https://git-scm.com/docs/git-clone)